lightspeed/modules/proc_login-vscode-extension.adoc

```diff
@@ -35,23 +31,26 @@ If you are using a cloud development environment at a domain unknown by Ansible
 == Procedure
 
 . Open the VS Code application.
-. Use one of the following ways to connect to the Ansible Lightspeed service. 
-** Using the *Connect* button:
-.. From the navigation menu, click the *Ansible* icon. 
-.. Under Ansible Lightspeed Login, click *Connect*.
-** Using the *Accounts* button:
-.. From the navigation menu, click the *Accounts* icon and select *Sign in with Ansible Lightspeed*.
+. Sign in using either the *Connect* button in the Ansible Lightspeed view or the *Accounts* menu.
```

Comment:
You should tag menu items with the menu tagging. This makes searching for menu selections easier, and will make conversion easier in the future. menu:Accounts[]

We also include the button tagging for buttons in the AAP library; however, the formatting still needs to be fixed by the DXS team because it prints the brackets, so it's up to you whether or not to change it.

btn:[Connect]

Reply:
Changed to menu:Accounts[], but retaining Connect button usage as there is an underlying formatting bug, and we don't know the completion time. Thank you.